html/ui-crisis.js

```diff
@@ -96,7 +96,9 @@ function crisisForm(messageId) {
 		event.stopImmediatePropagation();
 		event.preventDefault();
 		var reason = jQuery('input[name="reason"]').val();
-		if (reason.length > 0) {
+      if(reason === (undefined || "")){
```

Comment:
```suggestion
      if(!reason) {
```
Because `undefined`, `""` are equivalent to `false` inside an `if`